File: promblem solving/codeforces/A_Nearly_Lucky_Number.py
n=int(input())
k=str(n)
c=0
for i in k:
    # print(i)
    if(i=="4"or i=="7"):
        c=c+1
        
# print(c)
if(c==4 or c==7):
    print("YES")
else:
    print("NO")        

File: promblem solving/codeforces/A_Nearly_Lucky_Number.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
int n;
cin>>n;
int c=0;
while(n>0){
    if((n%10)==4 or (n%10)==7)
    {
        c=1;
        break;
    }
    else
    n=n/10;
}
if(c)
cout<<"YES"<<"\n";
else
cout<<"NO"<<endl;
return 0;
}